back/lib/shared/constants/constants.go

```diff
@@ -57,3 +57,9 @@ var SelectHas = map[string]string{
 const (
 	PaginationLimit int = 10
 )
+
+// PASSWORD AND SALT
+const (
+	SaltLength int    = 64
+	Chars      string = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890!@#$%"
```

Comment:
Hard coded salt 😳

Reply:
It's the characters use for creating the salt and not the salt itself (I agree that the comment above is not clear).

I have not found the equivalent of `fromCharCode` for Go